apps/crn-server/src/data-providers/contentful/analytics.data-provider.ts

```diff
@@ -52,7 +62,17 @@ export class AnalyticsContentfulDataProvider implements AnalyticsDataProvider {
             ),
             interestGroupPreviousMemberCount:
               (team.inactiveSince &&
-                team.linkedFrom?.interestGroupsCollection?.total) ||
+                getUniqueIdCount([
+                  ...(team.linkedFrom?.teamMembershipCollection?.items.flatMap(
+                    (item) =>
+                      item?.linkedFrom?.usersCollection?.items
+                        .filter(filterOutAlumni)
```

Comment:
if the team is inactive, does it matter if the user is alumni or not?

Reply:
I don't think that part of the expression is being evaluated in that case, when the team is inactive it should immediately break out of the expression at line 64

Reply:
Would it? if the team is inactive, `team.inactiveSince` is truthy and it would continue, right?
![Screenshot 2024-03-28 at 07 13 29](https://github.com/yldio/asap-hub/assets/16595804/f6bcbba2-86af-4cb2-8eca-40ade56c0b34)

Reply:
Sorry you're right I misunderstood. This is actually a bug and should be fixed in the follow up ticket ASAP-359